libr/core/linux_heap_glibc.c

```diff
@@ -21,6 +21,37 @@
 #define GHT_MAX UT64_MAX
 #endif
 
+
+static GHT GH(get_va_symbol)(RCore *core, const char *path, const char *symname) {
+	RBin *bin = core->bin;
+	GHT vaddr = GHT_MAX;
+	RListIter *iter;
+	RBinSymbol *s;
+	const char *bin_file = bin->file;
+	ut64 baddr = r_config_get_i (core->config, "bin.baddr");
+	ut64 laddr = r_config_get_i (core->config, "bin.laddr");
+	int fd = r_io_fd_get_current (core->io);
+	int rawstr = bin->rawstr;
+
+	RBinOptions opt;
+	r_bin_options_init (&opt, -1, 0, 0, false);
+	r_bin_open (bin, path, &opt);
+	RList *syms = r_bin_get_symbols (bin);
+
+	if (syms) {
```

Comment:
no need for this check, you can just use `r_list_foreach` directly as it already checks for NULL.